features-json/jspi.json

```diff
@@ -0,0 +1,635 @@
+{
+  "title": "JavaScript Promise Integration",
+  "description": "Bridges the gap between synchronous WebAssembly applications and asynchronous Web APIs",
+  "spec": "https://github.com/WebAssembly/js-promise-integration",
+  "status": "wd",
```

Comment:
technically speaking, this feature hasn't reached our W3C WD yet. It's in the CG stage 3 of standardization, and after hit reaches stage 4 it can be shipped (and then we'll merge it into the WD). Not sure what the best status for that is.